smoketest/configs/dialup-router-complex

```diff
@@ -1361,6 +1361,7 @@ service {
             listen-address 172.16.254.30
             listen-address 172.31.0.254
             negative-ttl 60
+            timeout 5000
```

Comment:
Please remove this line - this config is from `1.3-beta-202101091250` and thus does not have this knob. Happy to merge this after removal of this line!